golang/internal/helper/image/image.go

```diff
@@ -111,3 +130,112 @@ func GetRegistryURLProto(registry *string, registryAuth *agent.RegistryAuth) str
 		return ""
 	}
 }
+
+func GetImageByReference(ctx context.Context, reference string) (types.ImageSummary, error) {
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		log.Fatal().Err(err).Send()
```

Comment:
I know it is used as this, but being more like a library element here, it would be better to return the error, as done in other functions in this file. That also makes zerolog import unnecessary here.

Reply:
You are right 👍